### ISSUE TITLE
Error messages can't get through

### ISSUE THREAD

**Issue by jussiarpalahti** [charto/charto-frontend#1]
It seems that System.js somehow interferes with error messaging and blocks the actual error:

<img width="708" alt="System.js error message" src="https://cloud.githubusercontent.com/assets/11941245/14579651/81542218-03b9-11e6-8240-1128b9b9f9f3.png">

Steps to reproduce:
- npm install react react-dom
- Change app.js to code below
- npm start

``` javascript
var React = require('react');
var ReactDOM = require('react-dom');

var o = { key: "value" };

Component = React.createElement("div", null, 
    React.createElement("h1", null, o)
);

ReactDOM.render(Component, document.getElementById('map'));
```

I tried to set "production: false" in various places which seems to contain System.js config. Node process env as development didn't affect this. System.js seems to think its in production mode, but it still loads React libraries in non-minified pieces instead of one minified bundle.

**Comment by jjrv** [comment]
`node_modules/cbuild/process.js` defines `NODE_ENV` as `production`, changing that to `development` shows the original error message. The stack trace could still be better though...

**Comment by jjrv** [comment]
Latest cbuild in Github now has a `-d` flag. Adding it to the cbuild calls of package.json scripts in this repo will then fix the problem. I'll do some more testing before releasing the new cbuild to npm and then close this issue.

I just realized Chrome shows a decent stack trace from SystemJS if you click "..." instead of the down arrow.

**Comment by jjrv** [comment]
It could automatically set the -d flag if your `NODE_ENV` environment variable equals `development`.

**Comment by jussiarpalahti** [comment]
I'm not sure, but default with Webpack et al is development. This would make the initial experience better and provide more information. Only when (or if) moving to production are build scripts made for production.

Re: chrome dev tools: this explains why I got so different stack traces. I was sometimes clicking that (...) and sometimes somewhere else. It's possible, but pretty tedious to read React's error message this way.

**Comment by jussiarpalahti** [comment]
I couldn't get the cbuild upgraded and working with the dev mode.

I removed npm's older cbuild and did `npm install charto/cbuild`. It brought down the newer version, but it's missing the dist folder and thus can't function at all.

Instead I cloned the cbuild repo and managed to get it working **without** the -d flag by having Atom compile it. With the -d flag in use I got this:

```
Build error:
{ [Error: Cannot find module 'cbuild/process-dev.js' from '/Users/jussiarpalahti/tmp/charto-map-frontend']
  cause: [Error: Cannot find module 'cbuild/process-dev.js' from '/Users/jussiarpalahti/tmp/charto-map-frontend'],
  isOperational: true }
```

I haven't made any JS packages myself, so I'm completely out in the woods with this.

**Comment by jussiarpalahti** [comment]
Yes, now I got it. process-dev.js is same as process.js, but for development. I created it and ran bundle-reload. Now I get the correct error message. It seems the problem inducing statement in original source is lost. Though now I can't recall if these types of error messages worked correctly with Webpack or jspm...

Anyways, problem solved!

**Comment by jjrv** [comment]
Fixed in charto/cbuild@9aacccf and released v0.0.6 on npm.
